code_samples/api/migration/src/Command/MigrationCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Command;
+
+use Ibexa\Platform\Migration\MigrationService;
+use Ibexa\Platform\Migration\Repository\Migration;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class MigrationCommand extends Command
```

Comment:
Make Andrew happy :)
```suggestion
final class MigrationCommand extends Command
```